docker/worker/Dockerfile

```diff
@@ -24,6 +24,10 @@ VOLUME /var/lib/docker
 COPY docker/worker/Pipfile* /env/docker/worker/
 COPY setup.py Pipfile* README.md /env/
 COPY osv /env/osv
+# Remove the symbolic link
+RUN rm /env/osv/schema.json
```

Comment:
Can we avoid this if we just check out the submodule into the "osv" directory directly?

Reply:
We can, but osv-schema has a lot of unrelated files inside it which I wanted to avoid putting into worker. (i..e the docs project, the converting tooling (ghsa and debian)...etc).

Reply:
That seems OK, it's less than 1MB in total (not including the .git repo, which seems to be 2.6MB on my machine which is also not too bad), and we can move out the tooling stuff later if needed.

Reply:
Moved the entire folder into docker.